styles.js

```diff
@@ -1,14 +1,21 @@
 import { createGlobalStyle } from "styled-components";
 
-export default createGlobalStyle`
-  *,
-  *::before,
-  *::after {
+const GlobalStyle = createGlobalStyle`
+  :root {
+    --color-primary: #E5D3B3;
+    --color-header: #333333;
+    --color-footer: #987554;
+  }
```

Comment:
nice stuff. i like that

Reply:
that global variables for colors.
I also used for my border-radius, cause it was all the same